core/services/workflows/metering/metering.go

```diff
@@ -430,6 +465,8 @@ func (s *Reports) End(ctx context.Context, workflowExecutionID string) error {
 		return ErrReportNotFound
 	}
 
+	// if in metering mode, send to beholder
```

Comment:
Is this meant to be a TODO? Also should this logic happen all within `.SendReceipt`?